src/dynarec/arm64/dynarec_arm64_0f.c

```diff
@@ -421,9 +421,12 @@ uintptr_t dynarec64_0F(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                     GETGM(q0);
                     GETEM(q1, 0);
                     d0 = fpu_get_scratch(dyn);
-                    MOVI_8(d0, 0b10001111);
+                    d1 = fpu_get_scratch(dyn);
+                    MOVI_8(d0, 0b111);
                     VAND(d0, d0, q1);  // mask the index
+                    VSSHR_8(d1, q1, 7); // zero mask
                     VTBL1_8(q0, q0, d0);
+                    VBIC(q0, q0, d1);

```

Comment:
Why are you changing this one? Was it wrong before?

Reply:
I see.
So, I fixed the issue in the original code, and added your test in the test suite.
Place rebase you code with lastest code. You can remove your changes to both PSHUFB, the code is working correctly now (issue was only on the EMM PSHUFB, mask was too large).